volontulo/templates/organizations/organization_view.html

```diff
@@ -25,12 +25,16 @@
 
     <div class="form-horizontal">
         <div class="col-xs-offset-2 col-xs-10">
+            {% if allow_edit %}
             <a href="{% url 'organization_form' organization.name|slugify organization.id %}" class="btn btn-primary">Edytuj organizację</a>
```

Comment:
As we talk f2f - there is organization_form in urls definition. On changing organization_form to organization_edit I'll create another issue.